builtin/settingtypes.txt

```diff
@@ -343,6 +339,19 @@ texture_min_size (Minimum texture size) int 64 1 32768
 #    A restart is required after changing this option.
 fsaa (FSAA) enum 0 0,1,2,4,8,16
 
+[**Occlusion Culling]
+
+#    Type of occlusion_culler
+#    "loops" is the legacy algorithm with nested loops and O(N^3) complexity
```

Comment:
```suggestion
#    "loops" is the legacy algorithm with nested loops and O(n³) complexity
```

also shouldn't this indicate that the option is to be used in case of performance problems only?

Reply:
that's correct